.luarc.json

```diff
@@ -0,0 +1,13 @@
+{
+	"$schema": "https://raw.githubusercontent.com/sumneko/vscode-lua/master/setting/schema.json",
+	"diagnostics": {
+		"globals": {
+			"vim": true
+		},
+		"disable": {
+			"lowercase-global": true,
+			"missing-parameter": true
```

Comment:
This is reasonable; a lot of lua code automatically passes nil when no arguments specified.